frontend/src/views/SQR/Form.vue

```diff
@@ -65,7 +65,7 @@
                   Save draft
                 </v-btn>
                 <v-btn
-                  v-if="$can('submit', this.sqrForm)"
+                  v-if="$can('submit', this.abilityData)"
```

Comment:
You are using this data only once. I really feel it's an overhead to define the class and the property for that.

Reply:
To use it in the template it has to exist on the class. Maybe a plugin that holds all the data classes would replace this need?